client/src/components/DataList/index.js

```diff
@@ -1,20 +1,20 @@
 import React from "react";
-import { uniq } from "lodash";
 
-class Datalist extends React.PureComponent {
+class DataList extends React.PureComponent {
   render() {
     let options = [];
-
     if (this.props.options && this.props.processOptions) {
       options = this.props.processOptions(this.props.options);
     } else if (this.props.options) {
       options = this.props.options;
     }
 
+    const uniqueOptions = [...new Set(options)];
```

Comment:
This sort of uniqueness call, regardless of whether it's from `lodash/uniq` or using this Sindre Sorhus method, should not happen in the `render` call. Neither should `processOptions`, for that matter. You should process and then memoize these options either outside of this component (preferred) or in a lifecycle method, reducing the amount of unnecessary processing/uniqueness.